ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -262,7 +262,9 @@ public class Manager: NSObject, HTTPDownloadDelegate, UIGestureRecognizerDelegat
     if isKeymanHelpOn {
       inputViewController.showHelpBubble(afterDelay: 1.5)
     }
-
+    
+    //HERE getAssociatedLexicalModel(langID) and setLexicalModel
```

Comment:
If I'm interpreting this right, this would be where you activate an already-downloaded model, right?  Please refer to https://github.com/keymanapp/keyman/pull/1737/ for this - I've an implementation ready for you regarding this.

Reply:
Thanks. I'll replace it with calling your implementation when I get it. That's what I use //HERE for (it's like //TODO: but less formal)